kickstart/PXELinux_rhel_atomic.erb

```diff
@@ -0,0 +1,14 @@
+<%#
+kind: PXELinux
+name: RHEL Atomic PXE
```

Comment:
Sorry, I'm not sure I follow.  I mean, this template looks like the default Kickstart PXELinux file, can any atomic OS definition just use that existing template?